workflows/metagenomics/wf_theiameta_illumina_pe.wdl

```diff
@@ -47,6 +53,11 @@ workflow theiameta_illumina_pe {
       classified_out = "classified#.fastq",
       unclassified_out = "unclassified#.fastq"
   }
+  call krona_task.krona as krona_clean {
+    input:
+      kraken2_report = kraken2_raw.kraken2_report,
```

Comment:
you might want to use the kraken2_clean here

Reply:
This is a bug! Thank you so much for catching it! Do you have the availability to do a small fix @sage-wright?